source/AI.cpp

```diff
@@ -3548,8 +3548,7 @@ void AI::MovePlayer(Ship &ship, const PlayerInfo &player, Command &activeCommand
 			autoPilot = activeCommands;
 	}
 	bool shouldAutoAim = false;
-	if(Preferences::Has("Automatic aiming") && !command.Turn() && !ship.IsBoarding()
-			&& (Preferences::Has("Automatic firing") || activeCommands.Has(Command::PRIMARY))
+	if(Preferences::Has("Automatic aiming") && !command.Turn() && !ship.IsBoarding() && !Preferences::Has("Automatic firing")
```

Comment:
Wouldn't we also remove the check for if auto firing is active? The old behavior was "if auto aiming is on and either you're attempting to fire or auto firing is off." Seems like the entire part past the "and" should be removed, not just the check for if you're attempting to fire your primary. (It we certainly shouldn't be changing it to only activate auto aiming if auto firing is off, as then the two preferences wouldn't mix.)
```suggestion
	if(Preferences::Has("Automatic aiming") && !command.Turn() && !ship.IsBoarding()
```

Reply:
Yes, but the automatic firing also has automatic aiming anyway. So if both of them were on you'd be automatically aiming twice.

...I might try to fix that too.

Reply:
Never mind, you're right.